components/EditableForm.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <div>
+    <div
+      :class="[
+        'mx-auto',
+        'w-full',
+        'border-[2px]',
+        'border-like-cyan',
+        'rounded-[24px]',
+        'py-[24px]',
+        'px-[32px]',
+      ]"
+    >
+      <div class="flex items-center justify-between">
+        <Label v-if="message" :text="message" />
+        <Label
+          class="text-medium-gray"
+          content-class="underline cursor-pointer"
+          :text="guideMessage"
+          @click.native="isOpenDialog = true"
```

Comment:
should also set `tempMessage`(`messageInput`) = `message`? Otherwise, the user needs to retype the whole message if he/she wants to update

Reply:
I think we should only save the confirmed string.🤔

https://user-images.githubusercontent.com/75730405/202334788-e199561d-7b92-4d47-852f-4bf50fe69fed.mov

Reply:
OK